examples/HttpCodesApi/HtmlResponse.php

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
```

Comment:
**Question**
If I were using this framework I'd want this to be customizable, has any thought been given to that?

Reply:
You can extend `HtmlException` and override the `buildHtmlResponse`